docs-src/content/using-cubic-sdk/streaming.md

```diff
@@ -558,4 +558,50 @@ public static void streamMicrophoneAudio() {
 ```
 {{% /tab %}}
 
+{{% tab "Swift/iOS" %}}
+
+This example uses methods of the Client class to establish connection to Cubic server, list models, stream audio file and receive the transcription results. 
+You can call Client from your client view controller or any other class.
+
+``` swift
+import Cubic
+import GRPC
+
+class CubicExample {
+
+    let client = Client(host: "demo-cubic.cobaltspeech.com", port: 2727, useTLS: false)
```

Comment:
the demo server only works with TLS = true.

Reply:
Fixed here and also in `recognise.md`